resources/views/emails/applicants_before_three_days.blade.php

```diff
@@ -0,0 +1,21 @@
+<div>
+    <p>Dear HR,</p>
+    <br>
+    <p>I hope this email finds you well. This is to inform you that there are {{ $count }} applications that are in resume screening for more than 3 days. Please find the details of these applications below:</p>
+    <br>
+    <ol>
+        @foreach ($applicants as $applicant)
+            <li><strong>Name -</strong> {{ $applicant->name }}
+                <br> <strong>Email -</strong> {{ $applicant->email }}
+                <br> <strong>Phone -</strong> {{ $applicant->phone }}
+                <br> <strong>Applied on -</strong> {{ \Carbon\Carbon::parse($applicant->created_at)->format('j M Y') }}</li>
```

Comment:
I think this will also work. Please test with this once
```suggestion
                <br> <strong>Applied on -</strong> {{ $applicant->created_at->format('j M Y') }}</li>
```

Reply:
Can you share the email screenshot in Github comment

Reply:
![image](https://user-images.githubusercontent.com/92683476/222698470-3fc87c13-9240-49f9-a15b-69b7ee0dd8ce.png)

Reply:
<br> <strong>Applied on -</strong> {{ $applicant->created_at->format('j M Y') }}</li>. that code was not working. 
![image](https://user-images.githubusercontent.com/92683476/222733576-086c7695-55f2-418b-9918-837173485946.png)